NEWS.md

```diff
@@ -1,3 +1,14 @@
+# pmforest 0.2.0
+
+## New Features:
+
+* Adds two new arguments, `CI_bracket_open`, and `CI_bracket_close` for denoting the opening and closing intervals respectively (#32)
```

Comment:
grammar suggestion: ", `CI_bracket_open`, and `CI_bracket_close`" to ", `CI_bracket_open` and `CI_bracket_close`," (i.e. move second comma to end of group)